### ISSUE TITLE
CONNECTION REFUSED FOR 4 DAYS

### ISSUE THREAD

**Issue by Ps3ud-0nym** [iotaledger/wallet#689]
Using IOTA light wallet 2.5.4. What is going on? Tried every node, tried different weights and curls. 

WTF is going on?

**Comment by ac1dphr3aq** [comment]
0wn3d

**Comment by ScotchWiskey** [comment]
I have same problem. Connection refused. This is very stressful. Please help.

**Comment by angelithsc** [comment]
I can't enter in my IOTA Wallet. Attached photo about the problem. If you can help me, thanks in advance.
![wallet iota problem](https://user-images.githubusercontent.com/34388289/33792266-529b7bdc-dc71-11e7-8235-e16abc995591.jpg)

**Comment by Olga7777777** [comment]
Так же не могу войти в кошелек. Пишет нет соединения. Помогите пожалуйста.

**Comment by jonaszuberbuehler** [comment]
@Ps3ud-0nym Try using the nodes mentioned here https://github.com/iotaledger/wallet/issues/628#issuecomment-349716413. Worked for me and many others.

**Comment by Ps3ud-0nym** [comment]
@jonaszuberbuehler THANK YOU! It worked, 👍

**Comment by jonaszuberbuehler** [comment]
Cheers! Glad it worked.

**Comment by Ps3ud-0nym** [comment]
Now if only I could withdraw the rest of my balance from binance haha

**Comment by jonaszuberbuehler** [comment]
Is it still blocked due to network congestion?

**Comment by Ps3ud-0nym** [comment]
Yeah. Unfortunately.

**Comment by ScotchWiskey** [comment]
Yes that worked for me too so thank you so much Ps3ud-Onym for posting that. I am so relieved and happy to have my IOTA wallet back with all transactions confirmed and balance updated.  I have put the message out there so others will hopefully get their problems solved too. That's what we need to do is help each other.
